nautobot/extras/filters.py

```diff
@@ -98,6 +100,110 @@ class CreatedUpdatedFilterSet(django_filters.FilterSet):
     last_updated__lte = django_filters.DateTimeFilter(field_name="last_updated", lookup_expr="lte")
 
 
+class RelationshipFilter(django_filters.Filter):
```

Comment:
Should this be a subclass of `django_filters.MultipleChoiceFilter` instead? That might fix the `filter(... value)` issue mentioned in your comment below too.

Reply:
Great catch!